profiler/src/Demos/Website-AspNet/Website-AspNet.csproj

```diff
@@ -174,13 +158,7 @@
     <Content Include="fonts\glyphicons-halflings-regular.woff" />
     <Content Include="fonts\glyphicons-halflings-regular.ttf" />
     <Content Include="fonts\glyphicons-halflings-regular.eot" />
```

Comment:
We could tear these out too, they just increase the repo size

Reply:
remove all fonts and references to them, thanks